example/javamodule/12-jni/build.sc

```diff
@@ -0,0 +1,59 @@
+import mill._, javalib._
+
+object foo extends RootModule with JavaModule {
+  // Additional source folder to put C sources
+  def nativeSources = T.sources(millSourcePath / "native-src")
+
+  // Auto-generate JNI `.h` files from Java classes using Javac
+  def nativeHeaders = T {
+    os.proc("javac", "-h", T.dest, "-d", T.dest.toString, allSourceFiles().map(_.path)).call()
```

Comment:
We have `Jvm.jdkTool`, which tries to look up the tool in the currently used JRE/JDK.

Reply:
Thanks! I'll use that